source/appModules/explorer.py

```diff
@@ -251,15 +251,17 @@ def _get_TextInfo(cls):
 
 class WorkerW(IAccessible):
 	def event_gainFocus(self):
+		if ((winVersion.winVersion.major, winVersion.winVersion.minor) != (6, 1)):
+			return
 		# #6671: Normally we do not allow WorkerW thread to send gain focus event,
 		# as it causes 'pane" to be announced when minimizing windows or moving to desktop.
 		# However when closing Windows 7 Start Menu in some  cases
 		# focus lands  on it instead of the focused desktop item.
 		# Therefore redirect it to the child which is a desktop in this case.
 		if eventHandler.isPendingEvents("gainFocus"):
 			return
-		if self.firstChild:
-			self.firstChild.setFocus()
+		if self.simpleFirstChild:
+			self.simpleFirstChild.setFocus()

```

Comment:
Could you add note here about why `simpleFirstChild` is used instead of `firstChild`. What does this have to skip over?

Reply:
Done.